src/dotnet/CarbonAware.WebApi/Controllers/CarbonAwareController.cs

```diff
@@ -36,25 +36,44 @@ public CarbonAwareController(ILogger<CarbonAwareController> logger, ICarbonAware
     }
 
     [HttpGet("bylocations/best")]
-    public IEnumerable<EmissionsData> GetBestEmissionsDataForLocationsByTime([FromQuery(Name = "locations")] string[] locations, DateTime? time = null, DateTime? toTime = null, int durationMinutes = 0)
+    public async Task<IEnumerable<EmissionsData>> GetBestEmissionsDataForLocationsByTime([FromQuery(Name = "locations")] string[] locations, DateTime? time = null, DateTime? toTime = null, int durationMinutes = 0)
```

Comment:
Same comment as `GetEmissionsDataForLocationByTime`

Reply:
Modified in recent commit